assignments/objects.js

```diff
@@ -17,28 +17,53 @@ const example = {
   "gender": "F"
 }
 
+//constructor notation 
```

Comment:
this isn't how it's asked for in the example. Your way of doing it is interesting/different  lol. But I think you're making more work for youself than needed. 

But if it works, it works. Can't argue with results